components/permissions/android/java_sources.gni

```diff
@@ -1,4 +1,16 @@
 brave_components_permissions_java_sources = [
+  "//brave/android/java/org/chromium/chrome/browser/crypto_wallet/KeyringServiceFactory.java",
+  "//brave/android/java/org/chromium/chrome/browser/crypto_wallet/util/Blockies.java",
+  "//brave/components/permissions/android/java/src/org/chromium/components/permissions/BraveAccountsListAdapter.java",
   "//brave/components/permissions/android/java/src/org/chromium/components/permissions/BravePermissionDialogDelegate.java",
   "//brave/components/permissions/android/java/src/org/chromium/components/permissions/BravePermissionDialogModel.java",
 ]
+
+brave_components_permissions_java_deps = [
+  "//brave/components/brave_wallet/common:mojom_java",
```

Comment:
I don't like the idea of having permissions include wallet specific code like this. @yrliou we did something in your PR to break this dependency, but I don't remember off hand. I also want @goodov to review this

Reply:
that code is removed and we have a permission dialog in a browser process now